lib/hypgen/workflow.rb

```diff
@@ -23,13 +23,13 @@ def external_dependencies
 
     def params
       @params ||= {
-          :sections => @profile_ids.collect { |section_id| { :id => section_id.to_s } },
-          :startDate => @start_time,
-          :endDate => @end_time,
-          :experimentId => @experimentId,
-          :dapToken => Hypgen.config.dap_token,
-          :dapLocation => Hypgen.config.dap_url,
-          :namespace => Hypgen.config.namespace
+          sections: sections,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.